src/api-reference/request/v4.endpoints.schemas.markdown

```diff
@@ -0,0 +1,295 @@
+postedAmount|object|[Amount](#schema-amount)|The posted amount of the expected expense entry, in the transaction currency of the Request
+remainingAmount|object|[Amount](#schema-amount)|The remaining amount of the expected expense entry, in the transaction currency of the Request
+source|enum|-|The source that created the expected expense. It can be CASH_ADVANCE or TRAVEL_ALLOWANCE. <br>This field will be empty in any other case
+transactionAmount|object|[Amount](#schema-amount)|**Required** The amount of the expected expense entry, in the transaction currency paid to the vendor
+transactionDate|timestamp|[RFC 3339] yyyy-MM-dd'T'HH:mm:ss.SSS'Z'|**Required** The date of the transaction
+travelAllowance|object|[Travel Allowance](#schema-travelallowance)|The Travel allowance
+tripData|object|[Trip Data](#schema-tripdata)|The description of the trip
+vendor|object|[Vendor](#schema-vendor)|The vendor of the expected expense entry
+
+### <a name="schema-allocation"></a>Allocation
+
+Name|Type|Format|Description
+---|---|---|---
+allocationAmount|object|[Amount](#schema-amount)|The amount of the allocation calculated with the percentage value multiplied by the transaction amount on the expected expense. This amount is given in the transaction's currency and rounded to 8 decimals after decimal point.
+approvedAmount|object|[Amount](#schema-amount)|The amount of the allocation calculated with the percentage value multiplied by the approved amount on the expected expense. This amount is given in the user's currency and rounded to 8 decimals after decimal point.
+allocationId|string|-|The unique allocation identifier
+custom1 to custom20|object|[CustomField](#schema-customfield)|The details from the Custom fields. <br>These fields may not have data, depending on the configuration
+expenseId|string|-|The unique identifier of the expected expense associated with the allocation
+percentEdited|boolean|-|Whether the allocation percent has been edited
+percentage|number|-|The percentage of the total expected expense that this allocation represents
+postedAmount|object|[Amount](#amount)|The amount of the allocation calculated with the percentage value multiplied by the posted amount on the expected expense. This amount is given in the user's currency and rounded to 8 decimals after decimal point.
+systemAllocation|boolean|-|Whether the allocation is a system allocation, usually hidden from the user. If displayed to the user, should be read-only
+
+### <a name="schema-exchangerate"></a>Exchange Rate
+
+Name|Type|Format|Description
+---|---|---|---
+operation|string|-|Exchange rate operation. Possible values are: `MULTIPLY` or `DIVIDE`
+value|number|-|Exchange rate value
+
+### <a name="schema-expensetype"></a>Expense Type
+
+Name|Type|Format|Description
+---|---|---|---
+id|string|-|**Required** Unique identifier of the expense type
+name|string|-|Name of the expense type
+href|string|[RFC 3986]|Hyperlink to the resource for the expense type definition
```

Comment:
This was removed from the example, we need to remove it from the schema here too.